### DISCUSSION TITLE
Firmware repository

### DISCUSSION THREAD

**Discussion by mailiam** [tjko/riden-flashtool#9] [Show and tell]
Thanks @tjko for sharing such an effort.
I confirm it works flawlessly on my Mac.

For those looking for firmware, I could download latest firmware here: http://www.ruidengkeji.com/rdupdate/firmware/

**Comment by tjko** [comment]
There is also "custom" firmware by UniSoft that has lot of improvements over the official firmware.   

https://doug-gilbert.github.io/rd60xx-unisoft/rd60xx-unisoft.html

**Comment by pedro042** [comment]
It looks like the firmware repository has been moved or directory listing is not allowed. Does someine know new firmware URL or how to get it?

**Comment by pedro042** [comment]
I found it - https://www.eevblog.com/forum/testgear/ruideng-riden-rd6006-dc-power-supply/1350/
It's not moved, just directory listing is forbidden

Copied from the mentioned page (to not lost it)

_6006v1: http://www.ruidengkeji.com/rdupdate/firmware/RD60061/RD60061.json (preliminary version)
6006v2: http://www.ruidengkeji.com/rdupdate/firmware/RD60062/RD60062.json (normal version)
6012: http://www.ruidengkeji.com/rdupdate/firmware/RD60121/RD60121.json
6018: http://www.ruidengkeji.com/rdupdate/firmware/RD60181/RD60181.json
6024: http://www.ruidengkeji.com/rdupdate/firmware/RD60241/RD60241.json
6030: http://www.ruidengkeji.com/rdupdate/firmware/RD60301/RD60301.json
6006P: http://www.ruidengkeji.com/rdupdate/firmware/RD60065/RD60065.json
6012P: http://www.ruidengkeji.com/rdupdate/firmware/RD60125/RD60125.json
6018P: http://www.ruidengkeji.com/rdupdate/firmware/RD60185/RD60185.json_

**Comment by lorf** [comment]
RK6006: http://www.ruidengkeji.com/rdupdate/firmware/RK60066/RK60066.json
RK6006H: http://www.ruidengkeji.com/rdupdate/firmware/RK60067/RK60067.json
